lib/sedna/README.md

```diff
@@ -1,2 +1,285 @@
 
-### Sedna Python SDK
+# Sedna Python SDK
+
+The Sedna Python Software Development Kit (SDK) aims to provide developers with a convenient yet flexible tool to write the Sedna applications.
+
+This document introduces how to obtain and call Sedna Python SDK.
+
+
+## Introduction
+
+Expose the Edge AI features to applications, i.e. training or inference programs.
+
+## Requirements and Installation
+The build process is tested with Python 3.6, Ubuntu 18.04.5 LTS
+
+```bash
+# Clone the repo
+git clone --recursive git@github.com:kubeedge/sedna.git
+cd sedna/lib
+
+# Build the pip package
+python setup.py bdist_wheel
+
+# Install the pip package 
+pip install dist/sedna*.whl
+
+```
+
+Install via Setuptools
+
+```bash
+python setup.py install --user
+```
+
+## Use Python SDK
```

Comment:
please add some background to reader understand these code better.

Reply:
Please follow #159 for more detail